lib/xcodeproj/config.rb

```diff
@@ -288,11 +287,9 @@ def merge_attributes!(attributes)
       @attributes.merge!(attributes) do |_, v1, v2|
         v1 = v1.strip
         v2 = v2.strip
-        v1_split = v1.shellsplit
-        v2_split = v2.shellsplit
-        if (v2_split - v1_split).empty? || v1_split.first(v2_split.size) == v2_split
+        if v1.start_with?(v2)
           v1
-        elsif v2_split.first(v1_split.size) == v1_split
+        elsif v1.start_with?(v1)
```

Comment:
I think v1 is always going to start with v1?

Reply:
reverted this commit, we didnt have sufficient test coverage anyways :(